### PR TITLE
graphQL integration testing

### PR DESCRIPTION
**Pull request by eyakauleva**

### CODE REVIEW

File: src/test/java/com/solvd/micro9/users/integration/graphql/GraphQLIT.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import org.springframework.context.annotation.Import;
+import org.springframework.graphql.execution.ErrorType;
+import org.springframework.graphql.test.tester.GraphQlTester;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+@GraphQlTest(GraphqlUserController.class)
+@Import(GraphQLITConfig.class)
+public class GraphQLIT {
```
Comment (h1alexbel) [inline src/test/java/com/solvd/micro9/users/integration/graphql/GraphQLIT.java:22]: I think you can rename this class to `GraphqlUserControllerIT` or `GraphqlUserControllerITCase`

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/test/java/com/solvd/micro9/users/integration/graphql/GraphQLIT.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                .entityList(User.class)
+                .get();
+        List<User> users = List.of(TestUtils.getUser());
+        assertUsersList(users, result);
+    }
+
+    @Test
+    void verifyUsersAreFondByCriteriaTest() {
+        int size = 10;
+        String query = "{ findByCriteria (criteria: {name: \"a\"}, size: " + size
```
[the remark this answers is not in the document]
Comment (eyakauleva) [reply src/test/java/com/solvd/micro9/users/integration/graphql/GraphQLIT.java]: fixed 02fd94b

File: src/test/java/com/solvd/micro9/users/integration/graphql/GraphQLIT.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+import org.springframework.boot.test.autoconfigure.graphql.GraphQlTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.graphql.execution.ErrorType;
+import org.springframework.graphql.test.tester.GraphQlTester;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+@GraphQlTest(GraphqlUserController.class)
+@Import(GraphQLITConfig.class)
```
[the remark this answers is not in the document]
Comment (eyakauleva) [reply src/test/java/com/solvd/micro9/users/integration/graphql/GraphQLIT.java]: fixed 02fd94b

File: src/test/java/com/solvd/micro9/users/integration/kafka/UserProducerIT.java [hunk 4]
```diff
@@ -26,10 +27,14 @@ class UserProducerIT extends KafkaTestcontainers {
...
     private static final String TOPIC = "syncMongoElastic";
 
+    @Autowired
+    private ObjectMapper objectMapper;
+
     @Autowired
     private UserProducer producer;
 
     @Test
+    @SneakyThrows
```
[the remark this answers is not in the document]
Comment (eyakauleva) [reply src/test/java/com/solvd/micro9/users/integration/kafka/UserProducerIT.java]: fixed 02fd94b

File: src/test/java/com/solvd/micro9/users/integration/graphql/GraphQLIT.java (hunk 1 above)
Comment (eyakauleva) [reply src/test/java/com/solvd/micro9/users/integration/graphql/GraphQLIT.java]: fixed 02fd94b

File: src/test/java/com/solvd/micro9/users/integration/graphql/GraphqlUserControllerIT.java [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+                .execute()
+                .path("data.getAllUsers[*]")
+                .entityList(User.class)
+                .get();
+        List<User> users = List.of(TestUtils.getUser());
+        assertUsersList(users, result);
+    }
+
+    @Test
+    void verifyUsersAreFondByCriteriaTest() {
```
Comment (h1alexbel) [inline src/test/java/com/solvd/micro9/users/integration/graphql/GraphqlUserControllerIT.java:40]: @eyakauleva In JUnit 5, It's not necessary, to add suffix `Test` to the method's signature

Comment (eyakauleva) [reply src/test/java/com/solvd/micro9/users/integration/graphql/GraphqlUserControllerIT.java]: fixed 83582da

**Review by h1alexbel — APPROVED** [review]
nice

**Pull request merged**